libcloud/compute/drivers/openstack.py

```diff
@@ -1588,7 +1588,7 @@ def ex_resize(self, node, size):
 
         :rtype: ``bool``
         """
-        server_params = self._create_args_to_params(node, size=size)
+        server_params = {'flavorRef': size.id}
```

Comment:
For consistency, we should also rename this to ``ex_resize_node``. I will make that change in trunk and leave ``ex_resize`` there for backward compatibility reasons.